src/Query.ts

```diff
@@ -1,42 +0,0 @@
-export class Query {
-    private query_buffer: Uint8Array;
-    private param_buffer: Uint8Array;
-    private param_cnt: number;
-
-    /**
-     * Create a new query with the base query set. You can now add (any) additional parameters.
-     * 
-     * @param query The base query
-     */
-    constructor(query: string) {
-        this.query_buffer = new Uint8Array(Buffer.from(query, 'utf-8'));
-        this.param_buffer = new Uint8Array();
```

Comment:
We need a `Query` object! This will allow adding multiple parameters, say in a loop for example.